Dockerfile.generator

```diff
@@ -0,0 +1,16 @@
+FROM fedora:rawhide
+ENV CWT_DIR=/tmp/container-workflow-tool
+RUN dnf install -y go-md2man \
+    make git python3-PyYAML \
+    python3-GitPython \
+    python3-requests-kerberos \
+    python3-bugzilla \
```

Comment:
Do we need these in Fedora? no bugzilla module, and iirc you worked around not having to use kerberos to access dist-git right?

Reply:
cwt runs with `--disable-klist` anyway

Reply:
Though I guess we would have to import it conditionally now I think about it. Lets just remove `python3-bugzilla` then